expected/function/query-expand/multiple-terms.out

```diff
@@ -3,23 +3,10 @@ CREATE TABLE synonyms (
   synonyms text[]
 );
 CREATE INDEX synonyms_term_index ON synonyms (term);
-DO LANGUAGE plpgsql $$
-BEGIN
-	PERFORM 1
-		WHERE current_setting('server_version_num')::int >= 120000;
-	IF FOUND THEN
-		INSERT INTO synonyms
-			VALUES ('Groonga', ARRAY['Groonga', 'Senna']);
-		INSERT INTO synonyms
-			VALUES ('Groonga', ARRAY['Mroonga', 'PGroonga', 'Rroonga']);
-	ELSE
-		INSERT INTO synonyms
-			VALUES ('Groonga', ARRAY['Mroonga', 'PGroonga', 'Rroonga']);
-		INSERT INTO synonyms
-			VALUES ('Groonga', ARRAY['Groonga', 'Senna']);
-	END IF;
-END;
-$$;
```

Comment:
This change is related to the following commit.
- https://github.com/pgroonga/pgroonga/commit/b5bba12cbbb3dbcbb3d5f2c50e434b48a43880b1